test/v1/server/document_test.go

```diff
@@ -451,6 +460,183 @@ func (s *DocumentSuite) TestInsert_MultipleRows() {
 	}
 }
 
+func (s *DocumentSuite) TestInsert_AutoGenerated() {
+	collectionName := "test_autogenerated"
+	s.testAutoGenerated(Map{
+		"schema": Map{
+			"title": collectionName,
+			"properties": Map{
+				"int_value": Map{
+					"type": "integer",
+				},
+				"pkey": Map{
+					"type":          "string",
+					"autoGenerated": true,
+				},
+			},
+			"primary_key": []interface{}{"pkey"},
```

Comment:
From 1.18 it can be written as `[]any{"pkey"}`, we can start using it.

Reply:
sure